cordova-lib/src/cordova/plugin.js

```diff
@@ -360,6 +360,12 @@ function getPluginVariables(variables){
 
 function getVersionFromConfigFile(plugin, cfg){
     var pluginEntry = cfg.getPlugin(plugin);
+    if (!pluginEntry) {
+        var oldStylePluginId = pluginMapper[plugin];
+        if (oldStylePluginId) {
```

Comment:
'oldStylePluginId' is not the correct name for what the registry-mapper (pluginMapper) returns. it returns the package name, which is the new way of doing things. so, I'd call it something like 'packageName' ..